Project01-Calculator/Vatsala010/index.html

```diff
@@ -0,0 +1,102 @@
+﻿<!DOCTYPE html>
+<html>
+    
+<head>
+    <meta charset="utf-8">
+    <title>Calculator</title>
+    <meta name="author" content="Vatsala Tandon">
+    <style>
+	body{
+  background : black;
+	}
+section{
+    margin-top: 50px;
+    padding: 20px;
+    width: 22%;
```

Comment:
I guess this will not give you the desired effect. Try resizing your window and you'll see.